eng/scripts/Automation-Sdk-Init.ps1

```diff
@@ -2,13 +2,13 @@
 [string] $dotnetInstallScriptVersion = "v1"
 
 function GetDotNetInstallScript() {
-    $installScript = Join-Path $RepoRoot 'dotnet-install.ps1'
+    $installScript = Join-Path $RepoRoot 'dotnet-install.sh'
```

Comment:
Presumably, this will only work on non-Windows machines. Why the change? The ps1 file will also run in the current session instead of shelling out to a separate process, which can often lead to issues with parameter passing (especially when quotes are involved).

Reply:
Yes, we will run this only on linux. Previous changes were giving me [these](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4266624&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5&l=93) errors because I wrote the commands for Windows machines. With the current changes everything is [up and running](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4273481&view=logs&j=7f026a02-f755-5ce4-425c-2d5df22c9d55&t=20b436ef-0412-5b55-7996-5294882c57b5&l=86).